config/doctrine/Assignment.orm.xml

```diff
@@ -21,9 +21,9 @@
                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                   xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping https://www.doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
     <entity repository-class="OAT\SimpleRoster\Repository\AssignmentRepository" name="OAT\SimpleRoster\Entity\Assignment" table="assignments">
-        <id name="id" type="integer" column="id">
-            <generator strategy="AUTO"/>
-            <sequence-generator sequence-name="assignments_id_seq" allocation-size="1" initial-value="1"/>
+        <id name="id" type="uuid">
```

Comment:
I think that you miss updating open api specification as well

Reply:
Fixed